freshmaker/lightblue.py

```diff
@@ -631,7 +631,8 @@ def find_container_images(self, request):
 
     def _set_container_repository_filters(
             self, request, published=True,
-            release_categories=conf.lightblue_release_categories):
+            release_categories=conf.lightblue_release_categories,
+            vendors=conf.lightblue_repo_vendors):
```

Comment:
Could you add `vendors` to the docstring?

Reply:
sure